packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2640,6 +2642,7 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   @override
   void debugFillProperties(DiagnosticPropertiesBuilder properties) {
     super.debugFillProperties(properties);
+    properties.add(FlagProperty('needsCompositing', value: _needsCompositing, ifTrue: 'needs compositing'));
```

Comment:
high level: does this need to be included in both the toStringShort and the debugFillProperties?
When displaying toStringDeep() both will be displayed which seems a bit redundant.